### DISCUSSION TITLE
Curation report, March 20–24, 2023

### DISCUSSION THREAD

**Discussion by ghost** [wikipathways/wikipathways-help#29] [curation-reports]
Hi!
Please, find below my first curation report, for March 20–24. 

On March 24, there were 27 recently changed pathways. 

Curation summary:
•	“Annotate nodes” tag added: [WP5346](https://classic.wikipathways.org/index.php/Pathway:WP5346), [WP5342](https://classic.wikipathways.org/index.php/Pathway:WP5342), [WP5343](https://classic.wikipathways.org/index.php/Pathway:WP5343).
•	Addition of: ontology term + “annotate nodes” tag + proposed correction of node names: [WP5345](https://classic.wikipathways.org/index.php/Pathway:WP5345).
•	History description needed: [WP5341](https://classic.wikipathways.org/index.php/Pathway_Talk:WP5341) (see discussion for this pathway), [WP5335](https://classic.wikipathways.org/index.php/Pathway:WP5335) (see discussion with the first author).
•	Further pathway description recommended: [WP2909](https://classic.wikipathways.org/index.php/Pathway:WP2909).
•	“Add references” tag ([WP5323](https://classic.wikipathways.org/index.php/Pathway:WP5323)) or “Add references” + “Annotate nodes” tags ([WP5334](https://classic.wikipathways.org/index.php/Pathway:WP5334)) added.
•	Approved pathways on which I did not propose any changes: [WP5153](https://classic.wikipathways.org/index.php/Pathway:WP5153), [WP5152](https://classic.wikipathways.org/index.php/Pathway:WP5152), [WP5180](https://classic.wikipathways.org/index.php/Pathway:WP5180), [WP4348](https://classic.wikipathways.org/index.php/Pathway:WP4348), [WP534](https://classic.wikipathways.org/index.php/Pathway:WP534), [WP5151](https://classic.wikipathways.org/index.php/Pathway:WP5151), [WP5078](https://classic.wikipathways.org/index.php/Pathway:WP5078), [WP5036](https://classic.wikipathways.org/index.php/Pathway:WP5036), [WP5285](https://classic.wikipathways.org/index.php/Pathway:WP5285), [WP4491](https://classic.wikipathways.org/index.php/Pathway:WP4491), [WP5275](https://classic.wikipathways.org/index.php/Pathway:WP5275), [WP5271](https://classic.wikipathways.org/index.php/Pathway:WP5271), [WP5320](https://classic.wikipathways.org/index.php/Pathway:WP5320). 
•	Other pathways for which the current quality tags, on March 24, seemed complete to me, and for which I did not propose any changes: [WP5324](https://classic.wikipathways.org/index.php/Pathway:WP5324), [WP5322](https://classic.wikipathways.org/index.php/Pathway:WP5322).
•	Welcome message sent to some new WikiPathways authors.

•	Other changes which I have not proposed (I am not sure whether they are needed), but which may help (please, verify): 
o	[WP5344](https://classic.wikipathways.org/index.php/Pathway:WP5344): there are 8 “TTN” elements within the pathway. Is this needed?
o	[WP5305](https://classic.wikipathways.org/index.php/Pathway:WP5305) (this pathway does not identify all elements within it, I guess this is already considered in the “Annotate nodes” tag).
o	[WP5329](https://classic.wikipathways.org/index.php/Pathway:WP5329): some node annotations (those referring to other pathways) do not contain a hyperlink. Would it be useful to add a hyperlink to those annotations too?
o	The visual aspect of some pathways seems complicated, unclear. Maybe reorganizing some elements and eliminating some colors would help. This is the case of: [WP2909](https://classic.wikipathways.org/index.php/Pathway:WP2909) and [WP5323](https://classic.wikipathways.org/index.php/Pathway:WP5323) (in this case, also, very small letters as compared to the size of the pathway).

I hope this is useful :) 
Thank you. All the best,
Elena

**Comment by khanspers** [comment]
Hi @ElenaDR1 ! 
Thank you for your report, all good changes and suggestions. A few comments:

- [WP5344](https://classic.wikipathways.org/index.php/Pathway:WP5344): After reviewing the [referenced paper](https://europepmc.org/article/MED/21257761), it seems like there should be only one TTN node (TTN is just one large protein). Added a Needs Work tag with a comment. 
- [WP5305](https://classic.wikipathways.org/index.php/Pathway:WP5305): Yes, it seems the unannotated nodes are unknown/hypothetical. Added Needs Work tag with note.
-  [WP5329](https://classic.wikipathways.org/index.php/Pathway:WP5329): Yes, it would be good to have pathway nodes annotated for the ones that correspond to existing pathways, added as comment on Needs Work tag.
- [WP2909](https://classic.wikipathways.org/index.php/Pathway:WP2909) and [WP5323](https://classic.wikipathways.org/index.php/Pathway:WP5323): Added a comment about improving layout. For WP5323, it is by the same author as WP5329 and I believe the author is planning to remove the coloring.

**Comment by ghost** [comment]
Hi!
Kristina, thank you very much for your answer concerning those pathways. I find this information very useful. Thanks to all of you for this opportunity, for your help, and your positive reactions and comments. It is motivating to know that you have found the curation report useful.
Best wishes,
Elena
